force-app/main/default/classes/SOQL.cls

```diff
@@ -143,15 +167,16 @@ public with sharing class SOQL {
      */
     public static List<String> getContactNamesByAccount(Id accountId) {
         // Create a list to hold the Contact LastNames
+        List<String> contactLastNamesList = new List<String>();
 
         // Query the Contacts
+        List<Contact> contactList = [SELECT Id, FirstName,LastName FROM Contact WHERE AccountId = :accountId];
         // Loop through each Contact and add the LastName to the list
-        //for () {
-        //    
-        //}
-
+            for (Contact eachContact : contactList) {
```

Comment:
Try doing inline SOQL for loop instead of using a separate variable for contactList.